### PR TITLE
Steam Fix

### PR DESCRIPTION
**Pull request by Gastonius**
(please don't merge just yet! More commits coming this way)

### CODE REVIEW

**Comment by limpkin** [comment]
any ETA?

**Comment by Gastonius** [comment]
There's a lot to review in this PR, final will contain no console.log outputs (of course)

**Comment by limpkin** [comment]
are  you available to talk?

File: chrome_extension/vendor/mooltipass/device.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ mooltipass.device.getStatus = function() {
         }
     }
 
-    if (! mooltipass.device.usingApp ) mooltipass.device.useApp();
+    if (! mooltipass.device.usingApp && !isSafari) mooltipass.device.useApp();
```
Comment (limpkin) [inline chrome_extension/vendor/mooltipass/device.js]: why was this added here? no !firefox?

File: chrome_extension/popups/js/popup_status.js [hunk 2]
```diff
@@ -5,49 +5,82 @@ var isSafari = typeof(safari) == 'object'?true:false;
...
-        var global = chrome.extension.getBackgroundPage();
-        mooltipass.website.chooseCredentialFields();
-        close();
+    mpJQ("#btn-select-credential-fields").click(function( e ) {
+        if ( isSafari ) {
+            e.preventDefault();
+            var global = safari.extension.globalPage.contentWindow;
+            global.mooltipass.website.chooseCredentialFields();
+        } else {
+            var global = chrome.extension.getBackgroundPage();
```
Comment (nelabidi) [inline chrome_extension/popups/js/popup_status.js]: unused variable 'global' ?
